### PR TITLE
Fix references to old and outdated specifications.

### PR DESCRIPTION
**Pull request by msporny**
This PR fixes old and outdated references to external specifications.

I went back and forth on whether or not references to VC-JWT and VC-DATA-INTEGRITY should be normative or informative. The argument for normative is: we are providing normative guidance to implementers to use at least one of those mechanisms. The argument for informative is to not have those dependencies in the spec, but then the argument becomes "then how do you accomplish the 'Verifiable' part of the spec?" (folks that remember history will note that informative references worked against us during the v1.0 work, and will likely work against us this time since we are normatively defining those mechanisms).

My suggestion is that we keep these normative until it looks like it might bite us (like, when trying to go to PR or REC)... and then re-evaluate then.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html" title="Last updated on Apr 22, 2023, 3:53 PM UTC (1013531)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1086/fdbf89e...1013531.html" title="Last updated on Apr 22, 2023, 3:53 PM UTC (1013531)">Diff</a>

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -2999,7 +2997,7 @@ <h3>Zero-Knowledge Proofs</h3>
 <a>holder</a> and <a>subject</a> through the use of selective disclosure of the
 <a>verifiable credential</a> values. Some other cryptographic systems which rely
 upon zero-knowledge proofs to selectively disclose attributes can be found in the
-[[?LDP-REGISTRY]] as well.
+Verifiable Credential Specifications Directory [[?VC-SPECS]] as well.
```
Comment (decentralgabe) [inline index.html:3000]: curious about the `?` - how can we normatively refer to this?

**Review by TallTed — APPROVED** [review]
Nothing bad leaps out at me...

**Review by dlongley — APPROVED** [review]

File: common.js [hunk 2]
```diff
@@ -14,14 +14,14 @@ var vcwg = {
       ],
       publisher: "University of California, Irvine."
     },
-    "VC-EXTENSION-REGISTRY": {
-      title: "Verifiable Credentials Extension Registry",
-      href: "https://w3c-ccg.github.io/vc-extension-registry/",
+    "VC-SPECS": {
```
Comment (Sakurann) [inline common.js:17]:
```suggestion
    "VC-SPECS-DIR": {
```

Maybe a minor point, but just `VC-SPECS` is pretty confusing... 
this change if accepted will impact multiple lines in index.html.

File: index.html [hunk 3]
```diff
@@ -2231,8 +2228,9 @@ <h3>Extensibility</h3>
           </li>
           <li>
 Support multiple types of cryptographic proof formats through the use of
-Data Integrity Proofs [[VC-DATA-INTEGRITY]] and a variety of signature suites
-listed in the Linked Data Cryptographic Suites Registry [[?LDP-REGISTRY]]
+Data Integrity Proofs [[VC-DATA-INTEGRITY]] and a variety of cryptographic
+suites listed in the Verifiable Credential Specifications Directory
+[[?VC-SPECS]].
```
Comment (OR13) [inline index.html:2233]: This is kinda using the vc-specs-dir as a registry for:

https://w3id.org/security#DataIntegrityProof

cryptosuites property.

File: index.html [hunk 4]
```diff
@@ -2231,8 +2228,9 @@ <h3>Extensibility</h3>
           </li>
           <li>
 Support multiple types of cryptographic proof formats through the use of
-Data Integrity Proofs [[VC-DATA-INTEGRITY]] and a variety of signature suites
-listed in the Linked Data Cryptographic Suites Registry [[?LDP-REGISTRY]]
+Data Integrity Proofs [[VC-DATA-INTEGRITY]] and a variety of cryptographic
```
Comment (OR13) [inline index.html:2231]:
```suggestion
JSON Web Tokens [[VC-JWT]] and Data Integrity Proofs [[VC-DATA-INTEGRITY]] and a variety of cryptographic
```

File: index.html (hunk 1 above)
Comment (OR13) [reply index.html:3000]: AFAIK, we can't.

File: index.html [hunk 5]
```diff
@@ -3482,7 +3480,7 @@ <h3>Proof Formats</h3>
 
         <ul>
           <li>
-Verifiable Credentials using JSON Web Tokens [[?VC-JWT]], and
```
Comment (OR13) [inline index.html:3485]: @msporny what is the reason for this change?

File: index.html [hunk 6]
```diff
@@ -1036,10 +1036,8 @@ <h3>Concrete Lifecycle Example</h3>
 Implementers that are interested in understanding more about the
 <code>proof</code> mechanism used above can learn more in Section <a
 href="#proofs-signatures"></a> and by reading the following specifications:
-Data Integrity [[VC-DATA-INTEGRITY]], Linked Data Cryptographic Suites
-Registry [[?LDP-REGISTRY]], and JSON Web Signature (JWS) Unencoded Payload Option
-[[RFC7797]]. A list of proof mechanisms is available in the Verifiable
-Credentials Extension Registry [[VC-EXTENSION-REGISTRY]].
+Data Integrity [[VC-DATA-INTEGRITY]] and the "Proofs" section of the Verifiable
```
Comment (OR13) [inline index.html:1039]:
```suggestion
Securing Verifiable
Credentials using JSON Web Tokens [VC-JWT], Data Integrity [[VC-DATA-INTEGRITY]] and the "Proofs" section of the Verifiable
```

**Review by OR13 — CHANGES_REQUESTED** [review]
Overall direction seems good, a few suggestions and questions.

File: common.js (hunk 2 above)
Comment (OR13) [reply common.js:17]: I agree with this comment, but this change will need to be made everywhere the reference is, and the spec title is what matters most, this is just a ref pointer.

File: index.html (hunk 6 above)
Comment (selfissued) [reply index.html:1039]: I agree with Orie's suggestion.

**Review by selfissued — CHANGES_REQUESTED** [review]
It's inappropriate to omit VC-JWT as a securing mechanism.

File: index.html (hunk 1 above)
Comment (msporny) [reply index.html:3000]: `?` means it's an informative reference... we can't normatively reference an informative document, which VC-SPECS is. So, as @OR13 said, we can't normatively reference VC-SPECS; it is :ghost:W3C FORBIDDEN:ghost:.

File: index.html (hunk 6 above)
Comment (dlongley) [reply index.html:1039]: I didn't think that VC-JWT used `<code>proof</code>`. If it does, then I agree it makes sense to reference it here. Otherwise, why would it be mentioned in a section about "Implementers that are interested in understanding more about the `<code>proof</code>` mechanism" ?

File: common.js (hunk 2 above)
Comment (msporny) [reply common.js:17]: Why is "VC-SPECS" confusing as a reference marker, rather than "VC-SPECS-DIR"... what does the "DIR" add that is important to a reader?

File: index.html (hunk 6 above)
Comment (msporny) [reply index.html:1039]: @OR13, @selfissued, and @mprorock ... as @dlongley noted, VC-JWT doesn't use `proof`, that's why the reference was removed. VC-JWT is referred to, normatively, in other sections of the specification such as:

* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#dfn-conforming-processor
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#proofs-signatures
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#example-usage-of-the-proof-property-on-a-verifiable-credential
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#media-types
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#proof-formats
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#base-context
* ... and so on

The proposed change makes no sense, VC-JWT doesn't use `proof`. This section is about `proof`.

File: index.html (hunk 5 above)
Comment (msporny) [reply index.html:3485]: To normatively reference VC-JWT a proof mechanism.

**Comment by msporny** [comment]
@selfissued and @mprorock wrote:
> It's inappropriate to omit VC-JWT as a securing mechanism.

@OR13, @selfissued, and @mprorock ... as @dlongley noted, VC-JWT doesn't use `proof`, that's why the reference was removed. 

VC-JWT is referred to, normatively, in other sections of the specification such as:

* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#dfn-conforming-processor
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#proofs-signatures
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#example-usage-of-the-proof-property-on-a-verifiable-credential
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#media-types
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#proof-formats
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#base-context
* ... and so on

The proposed change to add VC-JWT  as a usage of `proof` does not make sense. 

The assertion that VC-JWT is being omitted as a securing mechanism is provably false, given the list above.

I am requesting a re-review from each of you.

**Comment by msporny** [comment]
> @OR13, @selfissued, and @mprorock ... I am requesting a re-review from each of you (5 days ago).

Re-ping... it's been 5 days. If I don't hear from each of you by 3pm ET tomorrow, I'm merging this PR.

File: common.js (hunk 2 above)
Comment (OR13) [reply common.js:17]: The spec title is more important than an internal document pointer.

The issue is people will probably assume the document has more standing than it does, but imo it's not really our problem, that is W3C problem.

File: index.html (hunk 6 above)
Comment (OR13) [reply index.html:1039]: vc-jwt DOES use `proof`... https://w3c.github.io/vc-data-model/#proofs-signatures

Its just not super clear how, because of this:
Screen Shot 2023-04-27 at 12 41 23 PM

File: index.html (hunk 3 above)
Comment (OR13) [reply index.html:2233]: @brentzundel @Sakurann @msporny I worry a bit about this,  if we will encounter complaints in the future.

Feels like we should either omit [[VC-JWT]] and [[VC-DATA-INTEGRITY]] and just point to the registry, or we should omit the registry pointer and just point to them.

We are kinda creating a 2 class system here.

Comment (OR13) [reply index.html:2233]: We could probably file an issue to track this and move on with the PR.

File: index.html (hunk 6 above)
Comment (OR13) [reply index.html:1039]: We could probably file an issue for this, and merge the PR as is.

**Review by OR13 — CHANGES_REQUESTED** [review]
I am requesting issues be filed for the ongoing discussions, but I am ok with the content as is, once they are filed.

- https://github.com/w3c/vc-data-model/pull/1086#discussion_r1179506740
- https://github.com/w3c/vc-data-model/pull/1086#discussion_r1179505323

File: index.html (hunk 6 above)
Comment (OR13) [reply index.html:1039]: As a follow up, since `proof` is allowed in `vc+ld+json`, it is assumed to be allowed in JWTs... so we can't hand wave over this, and the spec text we have today is now totally out of date with the latest resolutions regarding media types and proof.

**Review by selfissued — APPROVED** [review]
This PR appears to be an improvement over what it replaces.

File: index.html (hunk 6 above)
Comment (msporny) [reply index.html:1039]: Ok, I'll do that. 

Perhaps that section should be titled: "Securing Verifiable Credentials", and then we talk about external proofs and embedded proofs, and normatively point to both securing format specifications.

Comment (msporny) [reply index.html:1039]: Issue raised to track the concern here: https://github.com/w3c/vc-data-model/issues/1104

**Review by mprorock — APPROVED** [review]
Significantly improved - worth moving forward with as long as we track issues as noted by @OR13

**Comment by msporny** [comment]
Excellent! Thanks @OR13, @selfissued, and @mprorock -- I've raised issues related to @OR13's concerns and have some ideas on how we can address them in a future PR. Merging this one so we can crawl one inch further.

**Comment by msporny** [comment]
Normative, multiple reviews, changes requested and made, issues raised to track concerns, no objections, merging.
